components/camel-cxf/src/main/java/org/apache/camel/component/cxf/DefaultCxfBinding.java

```diff
@@ -747,7 +747,10 @@ protected static Object getContentFromCxf(Message message, DataFormat dataFormat
             }
             
             if (dataFormat == DataFormat.POJO) {
-                answer = message.getContent(List.class);  
+                List<?> pojoMessageList = message.getContent(List.class);  
+                if (pojoMessageList != null && pojoMessageList.size() != 0) {
```

Comment:
I favour using isEmpty() as then the impl can be optimized for that, instead of having to know the exact number of elements.